Documents/8-Kigeme-Rwanda-Use-Case.md

```diff
@@ -1,6 +1,4 @@
-![c1b04cd81c7afefa111aeac4b966ad18](media/c1b04cd81c7afefa111aeac4b966ad18.png)
-
-**Humanitarian Engineering for Energy for Displacement (HEED)**
+### **Humanitarian Engineering for Energy for Displacement (HEED)**
```

Comment:
Shouldn't be a heading and should just be left bold otherwise it will come up as an item in the contents on this page https://heedproject.github.io/rert/